docs/release-notes/5.7.41-44.md

```diff
@@ -0,0 +1,70 @@
+# Percona Server for MySQL 5.7.41-44 (2023-02-28)
+
+<style>
+    table {
+        width=50%;
+        font-family: Poppins;
+    }
+    table td {
+        border: 0px;
+        padding: 8px;
+    }
+</style>
+
+| Release date | February 28, 2023 |
+|---|---|
+| Install instructions | [Install Percona Server for MySQL](https://www.percona.com/doc/percona-server/5.7/installation.html) |
+
+
+[Percona Server for MySQL](https://www.percona.com/software/mysql-database/percona-server) 5.7.41-44
+includes all the features and bug fixes available in [MySQL 5.7.41 Community Edition](https://dev.mysql.com/doc/relnotes/mysql/5.7/en/news-5-7-41.html) in addition to enterprise-grade features developed by Percona.
+
+Percona Server for MySQL is a free, fully compatible, enhanced, and open
+source drop-in replacement for any MySQL database. It provides superior
+performance, scalability, and instrumentation.
+
+Percona Server for MySQL is trusted by thousands of enterprises to provide
+better performance and concurrency for their most demanding workloads. It
+delivers more value to MySQL server users with optimized performance,
+greater performance scalability and availability, enhanced backups, and
+increased visibility.
+
+For paid [support](https://www.percona.com/services/support), [managed services](https://www.percona.com/services/managed-services) or [consulting services](https://www.percona.com/services/consulting), contact [Percona Sales](https://www.percona.com/about-percona/contact)
+
+For [training](https://www.percona.com/training), contact [Percona Training - Start learning now](https://learn.percona.com/contact-me)
+
+## Release highlights
+
+Percona has removed an Oracle patch for mysqldump that performed, at the beginning of the dump, a `FLUSH_TABLES_WITH_READ_LOCK` and `GTID_EXECUTED` because the patch required additional user privileges, even when the user does not use GTID-based replication. The following bugs based on this patch were submitted to Oracle:
```

Comment:
```suggestion
Percona has removed an Oracle patch for mysqldump that performed, at the beginning of the dump, a `FLUSH_TABLES_WITH_READ_LOCK` to get consistent `GTID_EXECUTED` because the patch required additional user privileges, even when the user does not use GTID-based replication. The following bugs based on this patch were submitted to Oracle:
```

Reply:
Added.